examples/react-cra/package.json

```diff
@@ -19,7 +19,6 @@
     "@types/react": "^17.0.11",
     "@types/react-dom": "^16.9.0",
     "@types/react-router-dom": "^5.1.7",
-    "node-sass": "^6.0.1",
```

Comment:
We don't need `node-sass` anymore, and it's a pain to install (requires the "right" version of python).